collections/Lat_liturg/MS_Lat_liturg_g_7.xml

```diff
@@ -52,167 +46,176 @@
                         </msItem>
                      </msItem>
                      <msItem xml:id="MS_Lat_liturg_g_7-item1-item3">
-                        <locus>(fol.51)</locus>
-                        <note>Penitential psalms</note>
+                        <locus>(fols. 51r–56v)</locus>
+                        <note>Seven penitential psalms</note>
+                        <rubric>Septem psalmi penitenciales</rubric>
                      </msItem>
                      <msItem xml:id="MS_Lat_liturg_g_7-item1-item4">
-                        <locus>(fol.56v)</locus>
+                        <locus>(fols. 56v–62v)</locus>
                         <note>Litany of the saints</note>
+                        <rubric>Sequitur Letania</rubric>
                      </msItem>
                      <msItem xml:id="MS_Lat_liturg_g_7-item1-item5">
-                        <locus>(fol.62v)</locus>
-                        <note>Psalms for vespers throughout the week</note>
+                        <locus>(fols. 62v–78v)</locus>
+                        <note>Psalms and Magnificat antiphons for vespers throughout the week</note>
+                        <rubric>Psalmi communes dominicis diebus per totum annum de tempore</rubric>
                      </msItem>
                      <msItem xml:id="MS_Lat_liturg_g_7-item1-item6">
-                        <locus>(fol.78v)</locus>
+                        <locus>(fols. 78v–82r)</locus>
                         <note>Compline</note>
+                        <rubric>Leccio ante completorium</rubric>
                      </msItem>
                      <msItem xml:id="MS_Lat_liturg_g_7-item1-item7">
-                        <locus>(fol.82)</locus>
+                        <locus>(fols. 82r–87v)</locus>
                         <note>Prime</note>
                      </msItem>
                      <msItem xml:id="MS_Lat_liturg_g_7-item1-item8">
-                        <locus>(fol.87v)</locus>
+                        <locus>(fols. 87v–91r)</locus>
                         <note>Terce</note>
                      </msItem>
                      <msItem xml:id="MS_Lat_liturg_g_7-item1-item9">
-                        <locus>(fol.91)</locus>
+                        <locus>(fols. 91r–94v)</locus>
                         <note>Sext</note>
                      </msItem>
                      <msItem xml:id="MS_Lat_liturg_g_7-item1-item10">
-                        <locus>(fol.94v)</locus>
+                        <locus>(fols. 94v–98r)</locus>
                         <note>None</note>
                      </msItem>
                      <msItem xml:id="MS_Lat_liturg_g_7-item1-item11">
-                        <locus>(fol.98)</locus>
+                        <locus>(fols. 98r–106v)</locus>
                         <note>Benedictions</note>
+                        <rubric>Benedictio super agnum</rubric>
```

Comment:
Sorry if this is stupid but just to double check, given the rubric below, that this is really 'agnum' and not 'aquam'